app/src/main/java/br/org/cesar/knot_setup_app/activity/login/LoginPresenter.java

```diff
@@ -51,14 +52,63 @@ public void doLogin(String password){
         this::onErrorHandler);
     }
 
+    private void getOpenthreadConfig(){
+        String bearer,ip;
+        bearer = dataManager.getInstance()
+                .getPersistentPreference()
+                .getSharedPreferenceString(context,"token");
+
+        ip = dataManager.getInstance()
+                .getPreference()
+                .getSharedPreferenceString(context,"ip");
+
+        this.port = dataManager.getInstance()
```

Comment:
Why getting the port again?
You just got it at `LoginPresenter(ViewModel viewModel, Context context)`

Reply:
Done